app/assets/javascripts/central.js

```diff
@@ -7,6 +7,13 @@ var ProjectSearchView = require('views/project_search_view');
 var ProjectVelocityView = require('views/project_velocity_view');
 
 import TourController from 'controllers/tour/TourController';
+import Pusher from 'pusher-js';
+
+var pusherApiKey = process.env.PUSHER_APP_KEY;
```

Comment:
does `process.env` exposes all ENV variables?

Reply:
Only the ones you use. It replaces the code for each env var similarly to what is done by C's `#define`.